bin/demuxmix.R

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env Rscript
+if (!require("BiocManager", quietly = TRUE))
+    install.packages("BiocManager",repos = "http://cran.us.r-project.org")
+is_demuxmix<-require("demuxmix")
+if (!require("demuxmix", quietly = TRUE))
+    BiocManager::install("demuxmix")
+
```

Comment:
please install biocmanager and demuxmix using bioconductor-demuxmix provided by conda